tools.md

```diff
@@ -11,36 +11,234 @@ mainmenu: true
 .README files are extremely important to have. They should provide all the information necessary for a new developer to jump into the project and start coding and deploying to staging/production. Here's a high-level boilerplate:
 
 ```
-# Name of project
-Short description
 
-# Team
-Name, Role, Email
+# [Project Name]
 
-# Setup
-How to setup local
-How to sync up local database from staging/production
-How to sync up files from staging/production
+## Table of Contents
 
-# Run
-How to run it locally
-How to run it in docker
+1. [Introduction](#introduction)
```

Comment:
@JagjitUvic  I'd usually be looking for a `Quick Start` section if I've used this repo before, but, don't remember how to get it running locally again. Something like 
1. run `npm install`
2. open a new terminal window, and run  `cd semantic && npx gulp build` to start styling
3. in original terminal window, run `npm run start`
4. visit `localhost:3000` in your browser